src/devtools/client/debugger/src/components/Editor/Breakpoints/Panel/Panel.tsx

```diff
@@ -87,8 +88,10 @@ function Panel({
   // If we were fully using concurrent APIs, updates to something like focus range or breakpoint would be done in a transition,
   // which would expose an "is pending" flag that we could use to show e.g. "Loading..." while we're updating breakpoints.
   // In this case, since we're using the deferred API for this, we have to calculate the "is pending" flag ourselves.
-  const isPending =
-    unsafeFocusRegion !== unsafeFocusRegionForSuspense || breakpoint !== breakpointForSuspense;
+
+  // Do a deeper equality check - seems like we sometimes have breakpoints with equivalent contents
+  const breakpointsAreEqual = isEqual(breakpoint, breakpointForSuspense);
+  const isPending = unsafeFocusRegion !== unsafeFocusRegionForSuspense || !breakpointsAreEqual;
```

Comment:
Poking at the replay, it does seem like the values are _deep_ equal, but not _reference_ equal, and I haven't dug far enough to see exactly why and how that's happening.

Reply:
> I haven't dug far enough to see exactly why and how that's happening.

I think maybe we should in this case. My understanding is that this shouldn't happen (at least not due to `useDeferredValue`) and it would be good to understand _why_ it's happening (if it really is) so it doesn't bite us in other places unexpectedly.